allure-model/src/main/java/io/qameta/allure/model/StepResult.java

```diff
@@ -130,46 +128,6 @@ public StepResult setStage(final Stage value) {
         return this;
     }
 
-    /**
-     * Gets description.
-     *
-     * @return the description
-     */
-    public String getDescription() {
```

Comment:
in general changing that's the breaking change. There are some users that use our model as transport protocol for their internal system, so even if we have no use for it in both Allure Report & Allure TestOps but it still can be used by someone.

Is there any reason you want this removed?

Reply:
Ok, I just updated the javadoc